C/cohomCalg/bundled/patches/Makefile.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/Makefile b/Makefile
+index 1b462a6..24757f2 100644
+--- a/Makefile
++++ b/Makefile
+@@ -5,9 +5,11 @@
+ #########################################################################
+ 
+ # compilers & flags
+-CC        := g++
++CC        := cc
++CXX       := c++
+ CFLAGS    := -O3
+-LD        := g++
++CXXFLAGS    := -O3 -std=c++11
```

Comment:
This isn't actually complaining, but if I remember correctly full support for C++11 came with GCC 4.9, so you may want to change the preferred GCC version to 5 just to be safe.